File: 백준/Bronze/29986. Amusement Park Adventure/Amusement Park Adventure.cc
#include <iostream>

using namespace std;
int n, h;
int a[201];
int main()
{
    cin >> n >> h ;
    int count = 0;
    for (int i = 1; i<=n;i++){
        cin >>a[i];
        if (h >= a[i]){
            count+=1;
        }
    };

    cout << count << endl;
    return 0;
}

File: 백준/Bronze/29986. Amusement Park Adventure/Amusement Park Adventure.rs
use std::io::{stdin, stdout, BufRead, BufReader, BufWriter, Write};

fn main() {
    let mut reader = BufReader::new(stdin().lock());
    let mut writer = BufWriter::new(stdout().lock());

    let mut count = 0;
    let mut input = reader.lines();
    if let Some(Ok(line)) = input.next() {
        let (n, h) = {
            let nh = line
                .split_whitespace()
                .map(|x| x.parse().expect("error"))
                .collect::<Vec<u64>>();
            (nh[0], nh[1])
        };
        let mut count = 0;

        if let Some(Ok(line)) = input.next() {
            let v = line
                .split_whitespace()
                .map(|x| x.parse().unwrap())
                .collect::<Vec<u64>>();

            for i in v {
                match i {
                    i if i <= h => {
                        count += 1;
                    }
                    _ => {}
                }
            }
        }
        writeln!(writer, "{}", count).unwrap();
    }
}
